layouts/section/posts.html

```diff
@@ -16,7 +16,8 @@ <h1 class="text-3xl">
 
   <div class="max-w-[65ch] mx-auto mt-8 lg:mt-12">
     <ul class="timeline timeline-vertical">
-      {{ range .Pages.GroupByDate "2006" }}
+      {{ $groupByDate := default "2006" .Params.group_by_date }}
```

Comment:
I prefer `groupByDate` to `group_by_date` because Hugo used the camelCase format in the front matter.

Reply:
Also, refer to https://gohugo.io/methods/pages/groupbydate/#layout-string, I think it's better to name as `groupLayout`?